lib/pure/asynchttpserver.nim

```diff
@@ -34,9 +34,8 @@
 ##
 ##     server.listen Port(5555)
 ##     while true:
-##       if server.shouldAcceptRequest(5):
-##         var (address, client) = await server.socket.acceptAddr()
-##         asyncCheck processClient(server, client, address, cb)
+##       if server.shouldAcceptRequest():
```

Comment:
We could also show both the simple example and the new one (removing the common code or factoring it)